src/bridge/vehicle_bridge.rs

```diff
@@ -179,6 +183,15 @@ impl<'a> VehicleBridge<'a> {
             })
             .wait()?;
 
+        // Initialize attachment
```

Comment:
Please add some comments on why you added the attachment and also on its format.